app/imageUpload/imageController.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var fs = require('fs');
+var imgur = require('imgur');
+var multer = require('multer');
+
+var upload = multer({
+  dest: 'uploads/',
+  limits: {
+    fileSize: 10000000 //20MB
```

Comment:
is this 20MB?

maybe you should create a variable (no magic numbers)
`var tenMegaBytes = 1000000`